src/main/java/bookstore/controller/BookController.java

```diff
@@ -26,27 +30,33 @@ public BookController(BookService bookService) {
     }
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books", description = "Get all available books")
+    public List<BookDto> getAll(Pageable pageable) {
```

Comment:
Please recheck if swagger works correctly without @ParameterObject for pagable.

```suggestion
    public List<BookDto> getAll(@ParameterObject Pageable pageable) {
```

Reply:
I thought It was how it should be displayed. Fixed.